lib/helpers.js

```diff
@@ -92,7 +92,8 @@ _beforeEach.givenModel = function(modelName, attrs, optionalHandler) {
 
   beforeEach(function(done) {
     if(modelName === '__USERMODEL__') {
-      modelName = this.userModel ? this.userModel : 'user';
+      modelName = this.userModel ? this.userModel : 'User';
```

Comment:
It's better to use `loopback.getModelByType(Loopback.User)` to lookup the actual user model.